gradle.properties

```diff
@@ -18,12 +18,10 @@ ANDROID_SUPPORT_VERSION=28.0.0
 ANDROIDX_VERSION=1.0.0
 GLIDE_VERSION=4.16.0
 ANDROID_GRADLE_VERSION=8.4.2
-NDK_VERSION=28.0.13004108
+NDK_VERSION=21.1.6352462
```

Comment:
I bumped the version to the newest one, but the original repo bumped it to 21.xx. Let's use the same version as the original source.